app_dart/test/request_handlers/update_branches_test_data.dart

```diff
@@ -0,0 +1,125 @@
+// Copyright 2021 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+const List<int> stderrArray = [];
+const List<int> stdoutArray = [
```

Comment:
Putting bytes as test data makes it hard to reason what's going on. Instead, use something like

`'my stdout'.codeUnits`